src/main/java/teammates/ui/controller/InstructorEditInstructorFeedbackPageAction.java

```diff
@@ -0,0 +1,81 @@
+ * {@code FeedbackSubmissionEditPageData} will be generated and the page requested will be loaded.
+ */
+public class InstructorEditInstructorFeedbackPageAction extends Action {
+
+    @Override
+    protected ActionResult execute() throws EntityDoesNotExistException {
+        String courseId = getRequestParamValue(Const.ParamsNames.COURSE_ID); 
+        String feedbackSessionName = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
+        String instructorUnderModerationEmail = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_MODERATED_PERSON);
+        String moderatedQuestionNumber = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_MODERATED_QUESTION);
+
+        Assumption.assertNotNull(String.format(Const.StatusMessages.NULL_POST_PARAMETER_MESSAGE, 
+                                               Const.ParamsNames.COURSE_ID), 
+                                 courseId);
+        Assumption.assertNotNull(String.format(Const.StatusMessages.NULL_POST_PARAMETER_MESSAGE, 
+                                               Const.ParamsNames.FEEDBACK_SESSION_NAME), 
+                                 feedbackSessionName);
+        Assumption.assertNotNull(String.format(Const.StatusMessages.NULL_POST_PARAMETER_MESSAGE, 
+                                               Const.ParamsNames.FEEDBACK_SESSION_MODERATED_PERSON), 
+                                 instructorUnderModerationEmail);
+
+        new GateKeeper().verifyAccessible(
+                logic.getInstructorForGoogleId(courseId, account.googleId), 
+                logic.getFeedbackSession(feedbackSessionName, courseId),
+                false, Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
+        
+        InstructorAttributes instructorUnderModeration = logic.getInstructorForEmail(courseId, instructorUnderModerationEmail);
+
+        // If the instructor doesn't exist
+        if (instructorUnderModeration == null) {
+            throw new EntityDoesNotExistException("Instructor Email "
+                    + instructorUnderModerationEmail + " does not exist in " + courseId
+                    + ".");
+        }
+
+        FeedbackSubmissionEditPageData data = new FeedbackSubmissionEditPageData(account, student);
+
+        data.bundle = logic.getFeedbackSessionQuestionsBundleForInstructor(
+                feedbackSessionName, 
+                courseId, 
+                instructorUnderModeration.email);
+
+        Assumption.assertNotNull(data.bundle);
+        
+        data.setSessionOpenForSubmission(true);
+        data.setModeration(true);
+        data.setHeaderHidden(true);
+        data.setPreviewInstructor(instructorUnderModeration);
+        data.setSubmitAction(Const.ActionURIs.INSTRUCTOR_EDIT_INSTRUCTOR_FEEDBACK_SAVE);
+
+        if (moderatedQuestionNumber != null) {
+            data.setModeratedQuestion(moderatedQuestionNumber);
+        }
```

Comment:
Yes, the questionId should be used here. 

@kennho, can you check if it will require a large change if we try to use feedbackQuestionId here instead of question number?